### PR TITLE
Used Bootstrap Tour to create an inline tutorial

### PR DESCRIPTION
**Pull request by RamIyer1998**
Created an inline tutorial using Bootstrap Tour

### CODE REVIEW

File: www/bootstrap-tour/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,21 @@
+EditorConfig helps developers define and maintain consistent
```
Comment (JaimieMurdock) [inline www/bootstrap-tour/.editorconfig]: Add a `# `.

Perhaps `.editorconfig` should be added to `.gitignore`? Not everyone will use the same settings.

File: www/splash.mustache.html [hunk 2]
```diff
@@ -1,50 +1,62 @@
-<link rel="stylesheet" type="text/css" href="/css/splash.css" />
+<link rel="stylesheet" type="text/css" href="/css/splash.css">
+<link href="introjs/introjs.css" rel="stylesheet">
```
Comment (JaimieMurdock) [inline www/splash.mustache.html]: Since we moved to bootstrap-tour, remove `introjs`?

**Comment by RamIyer1998** [comment]
It appears that all I need is `bootstrap-tour.js`. I'll remove the other files.

**Comment by RamIyer1998** [comment]
@JaimieMurdock I implemented the requested changes, let me know if anything else needs work whenever you have the chance to look at it.

**Comment by JaimieMurdock** [comment]
For some reason the 32-bit Miniconda Appveyor installs aren't working. Examining.

**Pull request merged**